apigee_edge.module

```diff
@@ -108,6 +112,41 @@ function apigee_edge_user_format_name_alter(&$name, AccountInterface $account) {
   }
 }
 
+/**
+ * Prepares variables for Apigee entity templates.
+ *
+ * Default template: apigee-entity.html.twig.
```

Comment:
I can only see `apigee-entity-default.html.twig` template but I would prefer simply `apigee-entity.html.twig`.

Reply:
If we are going to keep one template for all edge entities, yeah let's go with `apigee-entity.html.twig`.

Reply:
The "default" suffix only tries to emphasize what I described in my next comment and in the header of the apigee-entity-default.html.twig. An entity type specific theme should not be implemented by adding entity type specific CSS, classes and template suggestions to the "apigee_entity_default" theme. It should be achieved by defining a theme for the entity_type. The "apigee_entity_default" theme is only used as a fallback solution if a theme/profile/module would not define an entity type specific theme.

Reply:
Should I rename it to `apigee_edge_fallback`? :)

Reply:
If multiple reviewers would expect the name to be `apigee-entity.html.twig`, maybe it would be worth using it since it would reduce confusion.

Reply:
I vote for `apigee-entity.html.twig` too. I think it's obvious that it's the default template without adding the `-default` suffix.

Reply:
Ok, I'll update the PR.